src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -8820,7 +8820,7 @@ public static String truncate(final String str, final int maxWidth) {
      * </p>
      * <ul>
      * <li>If {@code str} is less than {@code maxWidth} characters long, return it.</li>
```

Comment:
It turns out, this sentence isn't true since we don't return the same object. Would you mind fixing that as well.